lib/reverse_sentence.rb

```diff
@@ -1,4 +1,60 @@
 # A method to reverse the words in a sentence, in place.
+require 'pry'
 def reverse_sentence(my_sentence)
-  raise NotImplementedError
+  # binding.pry
+  if my_sentence == nil || my_sentence.length == 1 || my_sentence.length == 0
+    return my_sentence
+  end
+  my_sentence = reverse_words(my_sentence)
+  # binding.pry
+  my_sentence = string_reverse(my_sentence, 0, (my_sentence.length - 1))
+  # binding.pry
+  return my_sentence
```

Comment:
Since all the string manipulations are being done in-place, lines 6 and 12 can simply be `return`